priv/site/_includes/hero.slime

```diff
@@ -12,24 +12,44 @@ header.Banner
 
   nav.nav
     = link @env, "Home", to: "/"
-    = link @env, "Info", to: "/#info"
-    .Marquee aria-hidden="true"
+    = link @env, "Schedule", to: "/schedule"
+    .Marquee.show aria-hidden="true"
       = for _ <- 1..2 do
         div
           = for _ <- 1..2 do
             span All proceeds are for charity
             span This is a carbon-neutral event
             span 1 attendee = 1 tree
             span 28.05.2021 on a screen near you
-
-    /* a Schedule */
     = link @env, "Speakers", to: "/speakers"
     = link @env, "Tickets", to: "/tickets"
 
-
   = include(@env, "_includes/mobile_nav.slime", alternate: true)
 
 div.Video aria-hidden="true"
   video autoplay=true muted=true loop=true  poster="<%= url_for("videos/hero.jpg") %>"
     source src="<%= url_for("videos/hero.webm") %>" type="video/webm"
   = responsive_image "videos/home-page-hero-mobile.jpg"
+
+javascript:
+
+  const nav = document.getElementsByClassName("nav")[0];
```

Comment:
That element doesn’t have an id, is it better to use id over class?

What is destructuring assignment I think I never used it?

nav is a constant because I don’t want to change is value